hamilton/plugins/vaex_extensions.py

```diff
@@ -26,6 +26,15 @@ def fill_with_scalar_vaex(
     return df
 
 
+@registry.with_columns.register(vaex.dataframe.DataFrame)
```

Comment:
Should we be registering it as `NotImplemented`? Or should the fact that we *don't* register it mean it's not implemented...